app/assets/javascripts/helpers/toMoment.test.js

```diff
@@ -1,9 +1,19 @@
+import {toMomentFromTime} from '../helpers/toMoment';
 import {toMoment} from './toMoment';
 
+
 it('#toMoment and then back to text', () => {
   expect(toMoment('12/19/2018').format('YYYY-MM-DD')).toEqual('2018-12-19');
   expect(toMoment('3/5/2018').format('YYYY-MM-DD')).toEqual('2018-03-05');
   expect(toMoment('1/15/18').format('YYYY-MM-DD')).toEqual('2018-01-15');
   expect(toMoment('01/5/18').format('YYYY-MM-DD')).toEqual('2018-01-05');
   expect(toMoment('01-5-18').format('YYYY-MM-DD')).toEqual('2018-01-05');
-});
\ No newline at end of file
+});
+
+describe('#toMomentFromTime', () => {
+  it('works as expected because local timezone is set as expected', () => {
```

Comment:
👍 Gotcha — so this will fail for me locally unless I'm running the tests on EST.

Reply:
Yep, that's the idea, so that if for some reason timezones are wonky and fail a bunch of other tests, this test will fail to try to help pinpoint that as the issue.